hikari/impl/rest.py

```diff
@@ -96,8 +99,6 @@
 _APPLICATION_JSON: typing.Final[str] = "application/json"
 _APPLICATION_OCTET_STREAM: typing.Final[str] = "application/octet-stream"
```

Comment:
While you are here, a bit of an unrelated change. Could you make these `sys.intern()`?

Reply:
Do we need to intern constant values?

Reply:
If we use them a lot, it is probably best (in this case these are used quite a lot). It gives it a faster lookup time and a bit more efficient

Reply:
Seems unnecessary to tag that into random prs though and cpython should be interning strings implicitly anyway